llumnix/arg_utils.py

```diff
@@ -227,7 +227,7 @@ def add_cli_args(
 
         parser.add_argument('--dispatch-policy',
                             type=str,
-                            choices=['balanced', 'load', 'queue', 'flood'],
+                            choices=['balanced', 'load', 'queue', 'flood', 'rr'],
```

Comment:
Please merge main and add discription for rr in arg_utils.py and arguments.md.

Reply:
add rr in arg_utils.py
no discription for arguments.md as other dispatch policy